plugins/modules/ndo_l3out_node_group_policy.py

```diff
@@ -0,0 +1,350 @@
+description:
+- Manage L3Out Node Group Policy on Cisco Nexus Dashboard Orchestrator (NDO).
+- This module is only supported on ND v3.1 (NDO v4.3) and later.
+author:
+- Sabari Jaganathan (@sajagana)
+options:
+  template:
+    description:
+    - The name of the template.
+    - The template must be an L3Out template.
+    type: str
+    aliases: [ l3out_template ]
+    required: true
+  l3out:
+    description:
+    - The name of the L3Out.
+    type: str
+    aliases: [ l3out_name ]
+    required: true
+  name:
+    description:
+    - The name of the L3Out Node Group Policy.
+    type: str
+    aliases: [ l3out_node_group_policy ]
+  description:
+    description:
+    - The description of the L3Out Node Group Policy.
+    type: str
+  node_routing_policy:
+    description:
+    - The name of the L3Out Node Routing Policy.
+    - Providing an empty string will remove the O(node_routing_policy="") from L3Out Node Group Policy.
+    type: str
+  bfd:
+    description:
+    - The Bidirectional Forwarding Detection (BFD) multi-hop configuration of the L3Out Node Group Policy.
+    - Providing an empty dictionary will remove the O(bfd={}) from the L3Out Node Group Policy.
+    type: dict
+    suboptions:
+      auth:
+        description:
+        - The BFD multi-hop authentication of the L3Out Node Group Policy.
+        type: str
+        choices: [ enabled, disabled ]
+        aliases: [ bfd_multi_hop_authentication ]
+      key_id:
+        description:
+        - The BFD multi-hop key ID of the L3Out Node Group Policy.
+        type: int
+      key:
+        description:
+        - The BFD multi-hop key of the L3Out Node Group Policy.
+        type: str
+    aliases: [ bfd_multi_hop ]
+  target_dscp:
+    description:
+    - The DSCP Level of the L3Out Node Group Policy.
+    type: str
+    choices:
+      - af11
+      - af12
+      - af13
+      - af21
+      - af22
+      - af23
+      - af31
+      - af32
+      - af33
+      - af41
+      - af42
+      - af43
+      - cs0
+      - cs1
+      - cs2
+      - cs3
+      - cs4
+      - cs5
+      - cs6
+      - cs7
+      - expedited_forwarding
+      - unspecified
+      - voice_admit
+  state:
+    description:
+    - Use C(absent) for removing.
+    - Use C(query) for listing an object or multiple objects.
+    - Use C(present) for creating or updating.
+    type: str
+    choices: [ absent, query, present ]
+    default: query
+notes:
+- The O(template) must exist before using this module in your playbook.
+  Use M(cisco.mso.ndo_template) to create the L3Out template.
+- The O(l3out) must exist before using this module in your playbook.
+  Use M(cisco.mso.ndo_l3out_template) to create the L3Out object under the L3Out template.
+- The O(node_routing_policy) must exist before using this module in your playbook.
+  Use M(cisco.mso.ndo_l3out_node_routing_policy) to create the L3Out Node Routing Policy.
+extends_documentation_fragment: cisco.mso.modules
```

Comment:
can you add a `seealso` section please?
```suggestion
  Use M(cisco.mso.ndo_l3out_node_routing_policy) to create the L3Out Node Routing Policy.
seealso:
- module: cisco.mso.ndo_template
- module: cisco.mso.ndo_l3out_template
- module: cisco.mso.ndo_l3out_node_routing_policy
extends_documentation_fragment: cisco.mso.modules
```

Reply:
Done @gmicol